src/opcodes.h

```diff
@@ -1,6 +1,8 @@
 #include "cpu.h"
 
 int hexToDecimalMode(unsigned char hex);
+void adcOpcode(unsigned int address, struct CPU *cpu, struct BUS *bus);
+void sbcOpcode(unsigned int address, struct CPU *cpu, struct BUS *bus);
```

Comment:
could you put this function in alphabetic order as well?

Reply:
Done!